src/plugins/saved_objects/public/index.ts

```diff
@@ -17,5 +17,9 @@
  * under the License.
  */
 
-export * from './saved_object_finder';
-export * from './saved_object_save_modal';
+export * from './save_modal';
```

Comment:
Please don't use * for API exports to avoid unwanted API leakage
(We at @elastic/kibana-app-arch team are working on getting rid of it)

Reply:
:+1: Done.